src/parser/ts/grammar.ne

```diff
@@ -92,14 +95,26 @@ const extractFunction = ([func, br, args, BR]) => {
   }
   return { func }
 }
+const extractUnaryOperator = ([operator, element]) => 
+  operator ? [operator[0], element] : element
+const brakedUnaryOperator = ([_, operator, element]) => {
+  const args = [element];
+  
+  if (operator) {
+  	args.unshift(operator);
+  }
+
+  return {func: '', args}
+}
 %}
 
 query -> _ trimmed _ {% d => d[1] %}
 
 trimmed -> compositeElement {% id %}
   | function {% id %}
 
-function -> unary br arithmethicElements BR {% extractFunction %}
+function -> br unary_operator element BR {% brakedUnaryOperator %}
+  | unary br arithmeticElements BR {% extractFunction %}
```

Comment:
`unary br arithmeticElements BR` already includes element in brackets `(element)`
doesn't it work without line 116?

Reply:
no, cause it fails to parse `(-ts{})`